webserver/webserver/src/main/java/io/helidon/webserver/ServerListener.java

```diff
@@ -238,6 +246,14 @@ private void suspend(boolean shutdownExecutors) {
         try {
             // Stop listening for connections
             serverSocket.close();
+            if (configuredAddress instanceof UnixDomainSocketAddress udsa) {
```

Comment:
Can this be abstracted so that this code resides in the unix-domain-socket module ?